app/src/main/java/org/pesmypetcare/mypetcare/services/PetManagerService.java

```diff
@@ -110,4 +112,21 @@
      * @param dateTime The dateTime
      */
     void deletePetWashFrequency(User user, Pet pet, DateTime dateTime);
+
+    /**
+     * Add a periodic event to pet.
+     * @param user The user that wants to add a periodic event
+     * @param pet The pet which the user wants to add a periodic event
+     * @param event The period event
+     * @param period The periodicity of the period event
+     */
+    void registerNewPeriodicNotification(User user, Pet pet, Event event, int period) throws ParseException, ExecutionException, InterruptedException;
+
+    /**
+     * Delete a periodic event of pet.
+     * @param user The user that wants to delete a periodic event
+     * @param pet The pet which the user wants to delete a periodic event
+     * @param event The period event that user wants to delete
+     */
+    void deletePeriodicEvent(User user, Pet pet, Event event) throws ParseException, ExecutionException, InterruptedException;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Line is longer than 120 characters (found 126).](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5411036)